pkgs/applications/audio/midisnoop/default.nix

```
@@ -0,0 +1,70 @@
+{
+  pkgs ? import <nixpkgs> { },
+}:
+
+pkgs.stdenv.mkDerivation {
+  pname = "midisnoop";
+  version = "master";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "surfacepatterns";
+    repo = "midisnoop";
+    rev = "master"; # This 'master' branch has not been updated in a long time (latest commit bc30f60 from 11 years ago).
+    sha256 = "sha256-XDpDfTq5sAHqeYk3j+HA9H0SGq5sawrLzdK2rqN9fv4="; # Will notify us if there is ever a change
```

Wrong version and source markers. Version should be formatted as `VERSION`, if it was set in a build manifest or tagged ON that commit, or `VERSION-unstable-YYYY-MM-DD`, if it's any other commit (in which case VERSION is taken either from the latest tag or the source, or is 0 if there's no versioning whatsoever).

Have you actually read the docs (last checklist item)? It's all in there. Also, have you looked at the manual, other package recipes? Don't check the actions you have not done
```suggestion
  version = "";

  src = fetchFromGitHub {
    owner = "surfacepatterns";
    repo = "midisnoop";
    rev = "";
    hash = "sha256-XDpDfTq5sAHqeYk3j+HA9H0SGq5sawrLzdK2rqN9fv4=";
```